controllers/configuration_controller.go

```diff
@@ -107,9 +142,13 @@ func (r *ConfigurationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		r.ProviderName = configuration.Spec.ProviderReference.Name
 	}
 	if err := terraformApply(ctx, r.Client, ns, configuration, applyJobName, tfInputConfigMapsName, r.ProviderName); err != nil {
-		return ctrl.Result{RequeueAfter: 3}, errors.Wrap(err, "failed to create/update cloud resource")
+		if err.Error() == MessageApplyJobNotCompleted {
```

Comment:
In this case, an error should return and retry the apply. Right?

Reply:
I think this errors MessageDestroyJobNotCompleted and MessageApplyJobNotCompleted， just mean the job is not completed. If these errors occur, we just retry, don't need to return.